tests/unit/fetchGrantedAccess.test.ts

```diff
@@ -37,17 +37,19 @@ describe('fetchGrantedAccess', () => {
       .mockImplementationOnce(() => {
         return Promise.resolve({
           ok: true,
-          count: 1,
-          nextPage: 1,
+          count: 50,
```

Comment:
Probably more consistent to set `count: 1,` here as there is only one record in `MOCK_ORDER`.
wdyt?